tests/test_pydantic_factory.py

```diff
@@ -1,6 +1,5 @@
-from __future__ import annotations
```

Comment:
This shouldn't be removed and optional should be `float | None` I think? It will stringize the annotation so that it is compatible with 3.8+

Reply:
Apparently the tests in this repo dont use that except for one place, it isnt a big deal i guess.. should follow what the rest do